src/components/form.jsx

```diff
@@ -122,16 +122,16 @@ const Form = () => {
           {isLoading && <LoadingSpinner />}
 
           {data && (
-            <>
-              <div className="mt-2 text-purple font-bold text-xl ml-2 dark:text-deeppurple">
+            <div className="bg-ash shadow-lg border-2 border-deeppurple py-2 pl-2 rounded-lg mt-4">
+              <div className="mt-1 text-purple font-bold text-xl ml-2 dark:text-deeppurple">
```

Comment:
In light mode the definition is in `text-deeppurple`, but in dark mode it's back to `text-purple`. Since in dark mode the abbrevaition region is surrounded by  `bg-ash`, the text should remain `text-deepurple`. What do u think?
```suggestion
              <div className="mt-1 font-bold text-xl ml-2 text-deeppurple">
```

Reply:
opps sorry about that i will fix that and re-push

Reply:
i've corrected  this